match.go

```diff
@@ -178,23 +68,19 @@ func getEsc(chunk string) (r rune, nchunk string, err error) {
 	return
 }
 
-// Split splits path immediately following the final Separator,
+// Split splits the path p immediately following the final slash,
 // separating it into a directory and file name component.
-// If there is no Separator in path, Split returns an empty dir
+// If there is no slash in path, Split returns an empty dir
 // and file set to path.
 // The returned values have the property that path = dir+file.
-func Split(path string) (dir, file string) {
-	i := len(path) - 1
-	for i >= 0 && !isPathSeparator(path[i]) {
-		i--
-	}
-	return path[:i+1], path[i+1:]
+func Split(p string) (dir, file string) {
```

Comment:
Like with `Match` I think we could import the godoc from `path.Split` directly.

Reply:
This one is different because I had to rename the argument to prevent a clash with the path package. (Either that, or change the import, which would make the diff bigger.)

Reply:
If the text matches up but the only difference is `path p` then it’s fine. 👍 it was just hard to be sure about them without rigorous back-and-forth examination.